airflow/www/utils.py

```diff
@@ -147,6 +156,9 @@ def encode_dag_run(dag_run: DagRun | None) -> dict[str, Any] | None:
         'data_interval_end': datetime_to_string(dag_run.data_interval_end),
         'run_type': dag_run.run_type,
         'last_scheduling_decision': datetime_to_string(dag_run.last_scheduling_decision),
+        'external_trigger': dag_run.external_trigger,
+        'conf': conf,
+        'conf_is_in_json_format': is_json_format,
```

Comment:
```suggestion
        'conf_is_json': is_json_format,
```

Could we make the name less verbose?

Reply:
Yes good idea! Will change it.

Reply:
Fixed :)